### ISSUE TITLE
Bug: Non-Paged CQL Reads Gauge isn't working.

### ISSUE THREAD

**Issue by danielhe4rt** [scylladb/scylla-monitoring#2295]
*Installation details*
Dashboard Name: Scylla CQL
Scylla-Monitoring Version: 4.6, 4.7 (tested)
Scylla-Version: Any

### Problem

![Grafana Dashboard](https://github.com/scylladb/scylla-monitoring/assets/6912596/437a4c56-6a96-4a9d-9d0f-17fa1dc6bef1)

I was testing the Optimization Tab and saw that the "Non-Paged CQL Reads" gauge isn't working. After a couple hours debugging, I found out that the query is crashing due a `[[by]]` inside the statement that isn't present in any other gauge. 

To make sure I replicated it into other versions and also at Scylla Cloud and we still have the same problem. Maybe it is affecting other versions, but it would be really good to check it out.

Here's the current query: 
```promql

100 * (
    (
        sum(rate(
            scylla_cql_unpaged_select_queries{
                instance=~"[[node]]",
                cluster="$cluster", 
                dc=~"$dc", 
                shard=~"[[shard]]"
            }[1m]
        )) 
        - 
        sum(rate(
            scylla_cql_unpaged_select_queries_per_ks{
                ks="system", 
                instance=~"[[node]]", 
                cluster="$cluster", 
                dc=~"$dc", 
                shard=~"[[shard]]"
            }[1m]
        )) by ([[by]])
    )
    /
    sum(rate(
        scylla_cql_reads{
            instance=~"[[node]]",
            cluster="$cluster", 
            dc=~"$dc", 
            shard=~"[[shard]]"
        }[1m]
    ))
) OR vector(0)
```

The new query with the fix: 

```promql

100 * (
    (
        sum(rate(
            scylla_cql_unpaged_select_queries{
                instance=~"[[node]]",
                cluster="$cluster", 
                dc=~"$dc", 
                shard=~"[[shard]]"
            }[1m]
        )) 
        - 
        sum(rate(
            scylla_cql_unpaged_select_queries_per_ks{
                ks="system", 
                instance=~"[[node]]", 
                cluster="$cluster", 
                dc=~"$dc", 
                shard=~"[[shard]]"
            }[1m]
        )) // by removed
    )
    /
    sum(rate(
        scylla_cql_reads{
            instance=~"[[node]]",
            cluster="$cluster", 
            dc=~"$dc", 
            shard=~"[[shard]]"
        }[1m]
    ))
) OR vector(0)

```

Hope that it helps somehow.

**Comment by danielhe4rt** [comment]
Hey @amnonh! I'm running the LTS 4.7 and still having this issue. Can you verify if it was merged in the other versions?

![non-paged-gauge](https://github.com/scylladb/scylla-monitoring/assets/6912596/2b3f9ae8-3480-4ba9-bfd8-ff001f20feba)

**Comment by mykaul** [comment]
> Hey @amnonh! I'm running the LTS 4.7 and still having this issue. Can you verify if it was merged in the other versions?

4.7 isn't LTS (AFAIK) and the bug is fixed in 4.8 (not released yet).
